soc/riscv/riscv-privilege/miv/Kconfig.soc

```diff
@@ -11,5 +11,8 @@ config SOC_RISCV32_MIV
 	bool "Microsemi Mi-V system implementation"
 	select ATOMIC_OPERATIONS_C
 	select INCLUDE_RESET_VECTOR
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_A
```

Comment:
Sure about the A support? https://www.microsemi.com/document-portal/doc_download/1244850-mi-vrv32imc seems to imply no A support (but C).

Reply:
ditto